src/engine/strat_engine/thinpooldev.rs

```diff
@@ -14,7 +14,7 @@ use types::Sectors;
 
 pub struct ThinPoolDev {
     pub name: String,
-    dev_info: Option<DeviceInfo>,
+    pub dev_info: Option<DeviceInfo>,
     data_block_size: Option<Sectors>,
     pub low_water_mark: Option<DataBlocks>, // How close to full before we are worried?
```

Comment:
Please remove this comment. It is less than helpful for me, and I'm the only one who asked for comments at all. I'm preparing a PR against the dm docs: https://github.com/mulkieran/linux/pull/1, which should hopefully be more satisfactory.